packages/extension/src/view/devtools/components/cookies/cookiesListing/cookieTableContainer/editableComponents/editableTextInput.tsx

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies.
+ */
+import React, { useCallback, useState, useRef, useEffect, memo } from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import { useCookieStore } from '../../../../../stateProviders/syncCookieStore';
+import type { InfoType } from '@ps-analysis-tool/design-system';
+import { noop } from '@ps-analysis-tool/common';
+
+interface EditableTextInputProps {
+  info: InfoType;
+  changedKey: string;
```

Comment:
Need suggestion, how about `keyToChange` or something similar?
If works, please change to other components.

Reply:
fixed in [9de5a3d](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/278/commits/9de5a3df8410e59c636e0d8210f8ff5dea009c17)